src/Nri/Ui/Highlightable/V3.elm

```diff
@@ -370,6 +258,131 @@ fromMarkdown markdownString =
             |> List.indexedMap (\i highlightable -> { highlightable | index = i })
 
 
+{-| Same as `fromMarkdown`, but receives a list of inlines parsed markdown instead of a string
+-}
```

Comment:
would it make sense to you to add some more about your motivation here? For example…

```suggestion
{-| Same as `fromMarkdown`, but receives a list of inlines parsed markdown instead of a string.

You might want to use this if you are parsing highlightables out of lists or in other block contexts that [`fromMarkdown`](#fromMarkdown) does not support.
-}
```